docker-compose.yml

```diff
@@ -1,9 +1,23 @@
 #version: '3.4'
-
 networks:
   bibbox-default-network:
     external: true
 services:
+  
+  bibbox-sys-commander-angularenv:   # Angular for Frontend building
+    container_name: bibbox-sys-commander-angularenv
+    networks:
+      - bibbox-default-network
+    build:
```

Comment:
one todo would be if we could manage that no build is needed for the start up so that it is enough that the imaged are pulled from docker hub... but lets discuss and brainstorm about that another time

Reply:
well, the configuration and what the container needs to do is custom, so unless we upload the already built custom image to the docker hub, i don't see how we're going to do that.